service/src/main/java/org/mskcc/smile/cpt_gateway/service/impl/CPTServiceImpl.java

```diff
@@ -31,7 +31,21 @@
 public class CPTServiceImpl implements CPTService {
 
     private static final Pattern CPT_TOKEN_REGEX = Pattern.compile(".*token=(\\w+).*");
-    private static final Pattern IGO_REQUEST_ID_REGEX = Pattern.compile(".*requestId\":\"(\\w+).*");
+    private static final Pattern IGO_REQUEST_TRACKER_ID_REGEX = Pattern.compile(".*requestId\":\"(\\w+).*");
+    private static final Pattern IGO_REQUEST_ID_REGEX = Pattern.compile(".*igoRequestId\":\"(\\w+).*");
+    private static final Pattern IGO_SAMPLE_ID_REGEX = Pattern.compile(".*sampleName\":\"(\\w+).*");
+
+    @Value("${cpt.new_request_record_url}")
+    private String CPT_NEW_REQUEST_RECORD_URL;
+
+    @Value("${cpt.update_request_record_url:}")
+    private String CPT_UPDATE_REQUEST_RECORD_URL;
+
+    @Value("${cpt.update_sample_record_url:}")
+    private String CPT_UPDATE_SAMPLE_RECORD_URL;
+
+    @Value("${cpt.sample_status_record_url}")
```

Comment:
a couple of these prop's are configured differently (some have `:` at the end, others don't)

wasn't sure if this was intentional or not for these specific properties so I thought I would just point this out